src/modules/job-exec/job-exec.c

```diff
@@ -873,6 +883,10 @@ static int job_start (struct job_exec_ctx *ctx, const flux_msg_t *msg)
         jobinfo_decref (job);
         return -1;
     }
+
+    if (job->userid != getuid ())
+        job->multiuser = 1;
+
```

Comment:
Just noticed we have mixed use of `getuid()` and `geteuid()` to get the instance owner throughout the code base.  Since we're not manipulating privilege in flux-core, it seems like they are equivalent.  Do you concur?  (We should probably strive for consistency, but for another day mabye)

Reply:
I can't think of a reason why the real and effective uid would ever be different in the broker, except by accident. Given that, wouldn't calling `geteuid()` imply that this *could* be the case, and cause confusion (as it perhaps has here). Or is `geteuid()` more correct in this case?

Reply:
I  agree that `getuid()` is less laden with implication and so probably more appropriate.  I'll open an issue on the consistency.